prombench/Makefile

```diff
@@ -11,37 +11,87 @@ deploy: nodepool_create resource_apply
 clean: resource_delete nodepool_delete
```

Comment:
It'll be nice if we have a Makefile variable `PROVIDER` here, which gets set to one default provider (currently gke)